pkg/analytics/track.go

```diff
@@ -198,11 +198,10 @@ func TrackBuildTransientError(success bool) {
 }
 
 // TrackDeployStack sends a tracking event to mixpanel when the user deploys a stack
-func TrackDeployStack(success, isCompose, isOktetoRepo bool) {
+func TrackDeployStack(success, isCompose bool) {
 	props := map[string]interface{}{
-		"isCompose":          isCompose,
-		"deployType":         "stack",
-		"isOktetoRepository": isOktetoRepo,
+		"isCompose":  isCompose,
+		"deployType": "stack",
```

Comment:
shall we remove `TrackDestroyStack`?

Reply:
Perhaps rename to DeployCompose as is the only case is left?

Reply:
Done